src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -742,7 +742,7 @@ public static IEnumerable<object[]> GetAsync_IPBasedUri_Success_MemberData()
                             $"HTTP/1.1 200 OK\r\n" +
                             $"Date: {DateTimeOffset.UtcNow:R}\r\n" +
                             $"Set-Cookie: {cookie1.Key}={cookie1.Value}; Path=/\r\n" +
-                            $"Set-Cookie: {cookie2.Key}={cookie2.Value}; Path=/\r\n" +
+                            $"Set-Cookie : {cookie2.Key}={cookie2.Value}; Path=/\r\n" +
```

Comment:
You should add a comment to the end of this line to highlight the fact that having a space before the colon character is important and is really a separate, specific, test.  Otherwise, someone might get rid of that space later on.